core/modules/exact_match.cc

```diff
@@ -21,23 +21,24 @@ const Commands ExactMatch::cmds = {
     {"set_default_gate", "ExactMatchCommandSetDefaultGateArg",
      MODULE_CMD_FUNC(&ExactMatch::CommandSetDefaultGate), 1}};
 
-CommandResponse ExactMatch::AddFieldOne(
-    const bess::pb::ExactMatchArg_Field &field, struct EmField *f, int idx) {
+CommandResponse ExactMatch::AddFieldOne(const bess::pb::Field &field,
+                                        const bess::pb::FieldData &mask,
+                                        struct EmField *f, int idx) {
   f->size = field.size();
   if (f->size < 1 || f->size > MAX_FIELD_SIZE) {
     return CommandFailure(EINVAL, "idx %d: 'size' must be 1-%d", idx,
                           MAX_FIELD_SIZE);
   }
 
-  if (field.position_case() == bess::pb::ExactMatchArg_Field::kAttribute) {
-    const char *attr = field.attribute().c_str();
+  if (field.position_case() == bess::pb::Field::kAttrName) {
```

Comment:
That would be a much cleaner syntax but unfortunately protobuf doesn't expose those functions (for whatever reason it actually has them implemented but they are private?!) 
https://developers.google.com/protocol-buffers/docs/reference/cpp-generated#oneof